services/QuillLMS/app/views/pages/diagnostic_tool.erb

```diff
@@ -14,7 +14,7 @@
         Quickly determine which skills your students need to work on with our diagnostic activities. The diagnostics cover vital sentence construction skills and generate personalized learning plans based on your students&lsquo; performance.
         </p>
         <section class="tool-ctas diagnostic-tool">
-          <a href=<%= "/activity_sessions/anonymous?activity_id=-LKbzH_Er916zGjgHk5U" %> target="_blank" class="q-button cta-button bg-white text-darkred">Try a sample activity</a>
+          <a href=<%= "/activity_sessions/anonymous?activity_id=ffd22b0d-cafe-42fb-bc96-5d5a02a40a55" %> target="_blank" class="q-button cta-button bg-white text-darkred">Try a sample activity</a>
```

Comment:
Not necessary to do now, especially since we aren't planning on swapping these out again anytime soon, but I wonder if this is worth eventually making a constant somewhere so we don't have to update a bunch of different places if it changes.